src/beast/beast/http/src/beast_http_method.cpp

```diff
@@ -120,5 +120,6 @@ status_text (int status)
     return "Unknown HTTP status";
 }
 
-}
-}
+} // http
```

Comment:
Maybe. I'm not really concerned about optimizing this right now, because I'm not sure its even the right interface.